streamwebs_frontend/streamwebs/tests/models/test_macroinvert.py

```diff
@@ -0,0 +1,147 @@
+from django.test import TestCase
+
+from django.contrib.gis.db import models
+from django.apps import apps
+from itertools import chain
+
+from streamwebs.models import Macroinvertebrates
+
+
+class SiteTestCase(TestCase):
+
+    def setUp(self):
+        self.expected_fields = {
+            'school': models.CharField,
+            'date/time': models.DateTimeField,
+            'weather': models.CharField,
+            'site_name': models.ForeignKey,
+            'water_type': models.CharField,
+            'time_spent': models.PositiveIntegerField,  # in minutes
+            'num_people': models.PositiveIntegerField,
+            'riffle': models.BooleanField,
+            'pool': models.BooleanField,
```

Comment:
little nitpicking things:

- `date/time` should be something like `date_time`, i feel like that slash will cause headaches in the future
- I couldn't find what you meant by `water_type`on the form